src/css/list.scss

```
@@ -79,3 +79,8 @@ section:first-of-type {
   margin: 0;
   padding: 2rem 0;
 }
+
+// Scroll padding for sticky header
+:root.view--list {
+  scroll-padding-top: 3rem;
```

```suggestion
  scroll-padding-top: 2.75rem;
  scroll-behavior: smooth;
```

Without some sort of visual scroll of the item into place I'm concerned users may not know that there's stuff above it to look out for. Also, I think the exact "height" for the scroll offset is `2.75rem` instead of 3 (presuming it stays on 1 line)